src/generators/static/packageFileGenerator.ts

```diff
@@ -174,7 +201,29 @@ function regularAutorestPackage(
         "tsc && rollup -c rollup.config.js && npm run minify && mkdirp ./review &&  npm run extract-api",
       minify: `uglifyjs -c -m --comments --source-map "content='./dist/index.js.map'" -o ./dist/index.min.js ./dist/index.js`,
       prepack: "npm install && npm run build",
-      "extract-api": "api-extractor run --local"
+      pack: "npm pack 2>&1",
+      "extract-api": "mkdirp ./review && api-extractor run --local",
+      lint: "echo skipped",
```

Comment:
please wrap it with double quotes.

Reply:
@dw511214992 It will automatically be wrapped in double quotes at the time of generation. Ref: https://github.com/Azure/autorest.typescript/blob/main/test/integration/generated/additionalProperties/package.json#L3